sdk/include/platform/ibex/platform-dma.hh

```diff
@@ -0,0 +1,140 @@
+#pragma once
+
```

Comment:
This file looks like it’s describing the interface to the specific DMA controller for use in the compartment. I think this should be local to the DMA compartment, not a public header. The public header is the one that describes the public interface for other code to talk to any DMA engine, including this one: it should have the concept for a DMA engine and the interface to the compartment.